fleet_rental_check_vehicle/doc/index.rst

```diff
@@ -0,0 +1,16 @@
+============================
+ Fleet Rental Check Vehicle
+============================
+
+Usage
+=====
+
+* Open menu ``Sales / Configuration / Rental / Vehicle items to check`` 
+* Click on ``[create]`` button to create an item. For example, Oil
+* Create as many items as you need
+
+
+Uninstallation
+==============
+
+Optional section for uninstallation notes.
```

Comment:
![putin-delete-it](https://cloud.githubusercontent.com/assets/186131/16083559/dc369aea-332e-11e6-9553-ced20c2e25ff.jpg)